app/routes.js

```diff
@@ -3,192 +3,328 @@
+                    personalisation: {
+                        'application_reference': req.body.application_reference,
+                        'email_address': req.body.to,
+                        'token': req.body.token,
+                        'url': "http://localhost:8080/api/user"
+                    },
+                    reference: "email confirmation"
+                })
+                .then(response => console.log(response))
+                .catch(err => console.error(err))
+
+
+            console.log('Sending confirmation email');
         });
 
     // =====================================
     // SUBMISSION CONFIRMATION
     // =====================================
+
     router
 
-        .post('/confirm-submission', function(req, res) {
-
-            //construct email from post request JSON
-            var payload = new sendGrid.Email({
-                to      : req.body.to,
-                from    : configSendGrid.fromAddresses.test,
-                subject : 'Legalisation application confirmation ' + req.body.application_reference,
-                html    : templator.submissionConfirmationTemplate(req.body.application_reference, req.body.send_information, configSendGrid.urls.applicationServiceURL,req.body.user_ref, req.body.service_type)
-            });
-            console.info('Sending submission confirmation email');
-            return res.json(sendEmail(req,res, payload));
+        .post('/confirm-submission', function (req, res) {
+
+            console.log('confirm submission output', req.body);
+
+            var NotifyClient = require('notifications-node-client').NotifyClient
+
+            var notifyClient = new NotifyClient("conor_preprod_api_key-6c19e868-f026-4ff4-86ed-8effb112c0cc-f0323ce0-45bc-4cbb-8976-dc58b24a0f75")
+
+            if (req.body.user_ref !== "undefined" && req.body.user_ref !== null && req.body.user_ref !== "") {
+                if (req.body.service_type == 1) {//standard service
+
+                    notifyClient
+                        .sendEmail("1f0557da-4c66-4f05-95ff-1ea55daede9f", req.body.to, {
+                            personalisation: {
+                                'application_reference': req.body.application_reference,
+                                'email_address': req.body.to,
+                                'user_ref':req.body.user_ref
+                            },
+                            reference: "application submission standard"
+                        })
+                        .then(response => console.log(response))
+                        .catch(err => console.error(err))
+
+
+                    console.log('Sending submission confirmation email');
+                } else {
+                    (req.body.service_type == 2)
+                    {//premium and drop-off service
+
+                        notifyClient
+                            .sendEmail("2fa30eab-e847-4fc0-ae2b-1725923adb9a", "c.mcgandy@kainos.com", {
```

Comment:
Move this out into the config .env file, replace hard coded email address with request email address